app.js

```diff
@@ -16,3 +16,393 @@ const navSlide = () => {
+                <a
+                  href="https://www.w3.org/Style/CSS/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-1"
+                  >CSS</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-2"
+                  >HTML</a
+                >
+              </li>
+              <li>
+                <a
+                  href="
+ https://getbootstrap.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-3"
+                  >Bootstrap</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-4"
+                  >Ruby</a
+                >
+              </li>
+            </ul>
+            <a
+              href="#0"
+              class="btn-see-project active-btn-effect hover-btn-effect pop"
+            >
+              See project
+            </a>`;
+  mostRecentProjectWrapper.appendChild(img);
+  mostRecentProjectWrapper.appendChild(div);
+  portfolio.appendChild(mostRecentProjectWrapper);
+  portfolio.appendChild(projectsWrapperDiv);
+  portfolio.appendChild(descktopProjectsWrapperDiv);
+
+  for (let i = 0; i < 6; i += 1) {
+    if (projectsWrapperDiv) {
+      projectDiv = document.createElement('div');
+
+      projectDiv.classList.add('project', 'project-block');
+      projectDiv.innerHTML = `  <h3 class="project-title">${recentProjects[1].name}</h3>
+            <p class="project-details">
+             ${recentProjects[1].description}
+            </p>
+            <ul class="project-technology">
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="project-btn project-btn-1 active-btn-effect hover-btn-effect"
+                >
+                  ${recentProjects[1].workStack[0]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://getbootstrap.com/"
+                  class="project-btn project-btn-2 active-btn-effect hover-btn-effect"
+                  > ${recentProjects[1].workStack[1]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="project-btn project-btn-3 active-btn-effect hover-btn-effect"
+                  > ${recentProjects[1].workStack[2]}</a
+                >
+              </li>
+            </ul>
+            <a
+              href="#0"
+              class="see-project-btn active-btn-effect hover-btn-effect pop"
+              >See project</a
+            >`;
```

Comment:
- [ ] 👏👏 As I mentioned earlier, there are a couple of improvements that I'd like to suggest for this link:

- [ ] Currently, the popup only works for the first project button, but as per the project requirement, it should work for each project button. You need to ensure that clicking on any "Show Project" button will display the relevant details for that specific project.

- [ ] Additionally, the content of the popup should be dynamically generated based on the specific project details. Currently, the popup shows static data, but it should display the title, description, technologies, and links related to the specific project that the user clicks on. To achieve this, make use of the project array object that you have and dynamically showcase the project details in the popup when the corresponding button is clicked.

By making these adjustments, your popup functionality will be more interactive and user-friendly, allowing users to view specific project details when they click on the respective buttons. Keep up the good work, and happy coding! 🚀💻🌟